Assets/MixedRealityToolkit.Tests/PlayModeTests/Utils/TestButtonUtilities.cs

```diff
@@ -176,11 +173,20 @@ public static IEnumerator MoveHandAwayFromButton(Transform button)
             Vector3 p2 = button.transform.position;
             Vector3 p3 = button.transform.position - button.TransformDirection(ButtonTranslateOffset);
 
-            // Move the hand back
+            yield return MoveHand(p2, p3);
+        }
+
+        /// <summary>
+        /// Move the right hand from point 1 to point 2
+        /// </summary>
+        public static IEnumerator MoveHand(Vector3 p1, Vector3 p2)
```

Comment:
we have a test hand class that has simplified versions of move / show / etc - do we still need this util method?

Reply:
TestHand does not have a MoveTo of p1 to p2. The whole testhand wrapper and use of PlayModeTestUtilities.hand functions isn't a can I want to open in this PR